app/admin/nat/edit-result.php

```diff
@@ -80,10 +81,13 @@
 }
 
 # execute update
-if(!$Admin->object_modify ("nat", $_POST['action'], "id", $values))  { $Result->show("danger",  _("NAT $_POST[action] failed"), false); }
-else																 { $Result->show("success", _("NAT $_POST[action] successful"), false); }
-
+if(!$Admin->object_modify ("nat", $_POST['action'], "id", $values)) {
+    $Result->show("danger", _("NAT")." ".$_POST[action]." "._("failed"), false);
+}
+else {
+    $Result->show("success", _("NAT")." "$_POST[action]." "._("successful"), false); }
```

Comment:
Line 88:  missing string . 
```php
_("NAT")." "$_POST[action]
_("NAT")." ".$_POST[action]
```

Reply:
Line 87-Line 89:  too many closing brackets  (rogue } end of line 88)

Reply:
Right. Damn small characters. :smiley: 
Reworked.